adr/recipes/try_usage.py

```diff
@@ -25,7 +26,10 @@ def run(args):
     data = next(run_query('try_commit_messages', **query_args))['data']
 
     count = defaultdict(int)
-    count['total'] = len(data['message'])
+    try:
+        count['total'] = len(data['message'])
+    except KeyError:
+        return MissingDataError
```

Comment:
Close, you'll want to reraise this exception. You can do this with the `raise` keyword. Here's an old (but still good) article you can read if you want to learn more about re-raising exceptions:
http://www.ianbicking.org/blog/2007/09/12/re-raising-exceptions/index.html

But the main benefit of re-raising the exception (as opposed to just letting the original KeyError get raised), is that we can now be sure that this error was caused by ActiveData returning nothing, and not some other unrelated KeyError that happened to be raised elsewhere in the script.